packages/styling/tsconfig.json

```diff
@@ -1,25 +1,23 @@
 {
   "compilerOptions": {
-    "baseUrl": ".",
-    "outDir": "lib",
     "target": "es5",
+    "outDir": "lib",
     "module": "commonjs",
+    "lib": ["es5", "es2015.promise", "dom"],
     "jsx": "react",
     "declaration": true,
     "sourceMap": true,
     "experimentalDecorators": true,
     "importHelpers": true,
-    "moduleResolution": "node",
-    "noImplicitAny": false,
+    "noImplicitAny": true,
+    "noUnusedLocals": true,
     "strictNullChecks": true,
     "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
     "preserveConstEnums": true,
-    "lib": ["es5", "dom"],
+    "skipLibCheck": true,
```

Comment:
Revert this if it's not currently needed for this package--it's a bit of a sledgehammer which is probably preferable to avoid